x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client.ts

```diff
@@ -395,3 +407,89 @@ const conditions = {
     return [this.isFullyTranslated(), this.isNotInstalled()];
   },
 };
+
+const missing = (direction: estypes.SortOrder = 'asc') =>
```

Comment:
This function name is vague, can we call it `sortMissingValue` or something more descriptive?

Reply:
done